### PR TITLE
 DEVELOPER-1371 release JBT 4.3.0.Beta2 / JBDS 9.0.0.Beta2; fix invalid marketplace links, now that we've merged stuff into fewer entries; remove obsolete comment blocks

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

**Comment by nickboldt** [comment]
@fbricon @alexeykazakov PR updated to include both JBT and JBDS bits, but JBDS bits aren't quite in CSP / GG yet so don't push this yet.

File: _config/products.yml [hunk 1]
```diff
@@ -60,17 +60,40 @@ devstudio:
...
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2015-07-29
+        # no supported IS version at this time
+        #supported_devstudio_is_version: 9.0.0.Alpha1
+        documentation_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        release_notes_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        blog_announcement_url: /blog/beta2-for-mars.html
+        installers:
+          - name: Stand-alone Installer
+            url: http://www.redhat.com/j/elqNow/elqRedir.htm?ref=http://www.jboss.org/download-manager/content/origin/files/sha256/f4/f4ffd0b36a622229e4a8b5c4db4c8c4fac0fd09750b1ba2cba73c22c5c0f5f83/jboss-devstudio-9.0.0.Beta1-installer-standalone.jar
```
Comment (alexeykazakov) [inline _config/products.yml]: Wrong URL I guess. Points to Beta1.

File: _config/products.yml [hunk 2]
```diff
@@ -60,17 +60,40 @@ devstudio:
...
+            file_size: 485MB
+          - name: Installer With EAP 6.4.0
+            url: http://www.redhat.com/j/elqNow/elqRedir.htm?ref=http://www.jboss.org/download-manager/content/origin/files/sha256/32/329b93cc424feea6d396d9b9d69f508335c387a0780ee8fa880f5501919f6f65/jboss-devstudio-9.0.0.Beta1-installer-eap.jar
```
Comment (alexeykazakov) [inline _config/products.yml]: I see Beta1 in this URL too.

File: _config/products.yml [hunk 3]
```diff
@@ -60,17 +60,40 @@ devstudio:
...
+            file_size: 630MB
+        zips:
+          - name: Update site (including sources) bundle of JBoss Developer Studio
+            file_size: 507MB
+            url: http://www.redhat.com/j/elqNow/elqRedir.htm?ref=http://www.jboss.org/download-manager/content/origin/files/sha256/b7/b7289aa99840cd07fc7f51bf2deaa17f7ca9acaaf1a5b0e72adab85b286fe74d/jboss-devstudio-9.0.0.Beta1-updatesite-core.zip
```
Comment (alexeykazakov) [inline _config/products.yml]: Beta1 too

**Comment by nickboldt** [comment]
Good catch - forgot to s/Beta1/Beta2/ in those links. Fixed and force-push PR updated.

**Comment by nickboldt** [comment]
Verified: SHAs I put in the PR are the same as the ones that Chris just posted to DEVELOPER-1371.

However, seems there's another problem which will prevent releasing this -- see https://issues.jboss.org/browse/DEVELOPER-1371. Meanwhile, I guess you can write the blog and get that ready, @alexeykazakov  ?

**Comment by alexeykazakov** [comment]
@nickboldt the blog is ready and waiting for JBDS to be released https://github.com/jbosstools/jbosstools-website/pull/450

**Comment by alexeykazakov** [comment]
Pushed to master and production

**Pull request closed without merging**